sentences/en/homeassistant_HassGetState.yaml

```
@@ -3,33 +3,51 @@ intents:
   HassGetState:
     data:
       - sentences:
-          - (do you know|tell me|<what_is>) [the [current] (state|value) of] <name> [in <area>]
-        response: one
-        excludes_context:
-          domain:
-            - scene
-            - script
-      - sentences:
-          - is [the] [state of] <name> {on_off_states:state} [in <area>]
-        response: one_yesno
-        excludes_context:
-          domain:
-            - cover
+          - (is|are)[ there] any {on_off_domains:domain} {on_off_states:state}
+          - (is|are)[ there] any {on_off_domains:domain} ({on_off_states:state};in <area>)
+          - (is|are)[ there] any <area> {on_off_domains:domain} {on_off_states:state}
+          - (do you know|tell me) if there are any {on_off_domains:domain} {on_off_states:state}
+          - (do you know|tell me) if there are any {on_off_domains:domain} ({on_off_states:state};in <area>)
+          - (do you know|tell me) if there are any <area> {on_off_domains:domain} {on_off_states:state}
+        response: any
 
       - sentences:
-          - (is|are) [there] any {on_off_domains:domain} {on_off_states:state} [in <area>]
-          - (do you know|tell me) if there are any {on_off_domains:domain} {on_off_states:state} [in <area>]
-        response: any
+          - are [all ][[of ]the ]{on_off_domains:domain} [(switched|turned|set[ to]) ]{on_off_states:state}
+          - are [all ][of ]<area> {on_off_domains:domain} [(switched|turned|set[ to]) ]{on_off_states:state}
+          - are [all ][[of ]the ]{on_off_domains:domain} ([(switched|turned|set[ to]) ]{on_off_states:state};in <area>)
```

Use `<are_all>` to match up to `are all of the`. I also again think it is easier to read if we split out the `(foo;bar)` expansion again. Also, adding the area off query here.
```suggestion
          - <are_all> {on_off_domains:domain} [(switched|turned|set [to])] {on_off_states:state} [<in_area_floor>]
          - <are_all> <area_floor> {on_off_domains:domain} [(switched|turned|set [to])] {on_off_states:state}
          - <are_all> {on_off_domains:domain} ([(switched|turned|set [to])] <in_area_floor> {on_off_states:state}
          - <is> <area> {on_off_domains:domain} off
```